examples/replay.cc

```diff
@@ -14,21 +15,23 @@ class Replay : public sc2::ReplayObserver {
         sc2::ReplayObserver() {
     }
 
+    
+
+    //Called once when the game is started (or restarted)
     void OnGameStart() final {
-        const sc2::ObservationInterface* obs = Observation();
```

Comment:
Not sure why you deleted this. Now the logic in OnGameEnd won't output anything.